src/constants/schema.ts

```diff
@@ -48,6 +49,19 @@ export const taskCreationSchema = z.object({
     ),
 });
 
+/**
+ * The schema for the task creation form
+ */
+
+export const taskCreationSchema = z.object({
+  categoryId: z.string(),
+  title: z
+    .string()
+    .min(TASK_TITLE_MIN_LENGTH, minLength('Title', TASK_TITLE_MIN_LENGTH))
+    .max(TASK_TITLE_MAX_LENGTH, maxLength('Title', TASK_TITLE_MAX_LENGTH)),
+  description: z.string(),
```

Comment:
Could you leave a TODO that says this description needs a correct validation? Also, I'd appreciate it if you could share the necessary changes regarding schemas with the frontend team.

Reply:
I added todo comment in the method. Also, I am going to share this information with the members.

Reply:
Where did you add the comment...?

Reply:
Sorry, I accidentally wrote the comment in the wrong place, so I moved it to the correct location.